dashboard/test/models/user_test.rb

```diff
@@ -1563,6 +1563,18 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert_equal [], teacher.reload.permissions
   end
 
+  test 'assign_course_as_facilitator assigns course to facilitator' do
+    facilitator = create :facilitator
+    facilitator.course_as_facilitator = Pd::Workshop::COURSES.last
+    assert facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.last).any?
+  end
+
+  test 'delete_course_as_facilitator removes facilitator course' do
+    facilitator = create :facilitator
+    facilitator.delete_course_as_facilitator Pd::Workshop::COURSES.first
+    refute facilitator.courses_as_facilitator.where(course: Pd::Workshop::COURSES.first).any?
+  end
```

Comment:
This doesn't test anything. The facilitator has no CourseFacilitator entries initially. Make sure it starts with one that can be deleted :)

Also while not quite as clean for destroy, but similar to above, you can use 

```ruby
assert_difference 'Pd::CourseFacilitator', -1 do
  facilitator.delete_course_as_facilitator Pd::Workshop::COURSE_CSF
end
```